airflow/providers/google/cloud/operators/dataproc_metastore.py

```diff
@@ -340,6 +492,11 @@ def execute(self, context: 'Context') -> dict:
                 timeout=self.timeout,
                 metadata=self.metadata,
             )
+        self.xcom_push(
+            context,
+            key="service_conf",
+            value={"region": self.region, "service_id": self.service_id, "project_id": self.project_id},
+        )
```

Comment:
I'm wondering if it would make sense to add class/static method on operator link to handle this logic? Something like:
```py
DataprocMetastoreServiceLink.persist(task_instance=self)
```
Why? First this make it more obvious why we save this XCom (to use it in link), second it somehow nicely encapsulates the logic into link class. WDYT? CC @potiuk @josh-fell